django_lightweight_queue/middleware/transaction.py

```diff
@@ -2,16 +2,13 @@
 
 class TransactionMiddleware(object):
     def process_job(self, job):
-        if not connection.in_atomic_block:
-            transaction.set_autocommit(False)
+        transaction.atomic(savepoint=False).__enter__()
```

Comment:
Unsure about performance concerns, I think it's okay for now. Good to be aware of this though if we notice performance drops in the future .